applications/ShapeOptimizationApplication/custom_utilities/optimization_utilities.h

```diff
@@ -112,12 +112,12 @@ class OptimizationUtilities
     // ==============================================================================
     // General optimization operations
     // ==============================================================================
-    void compute_design_update()
+    void compute_control_point_update()
```

Comment:
Should be UpperCamalCase, see [Style Guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide#class-methods)

Reply:
There are more of those, haven't marked all of them